Jellyfin.Versioning/update-version

```diff
@@ -24,11 +24,19 @@
 #
```

Comment:
any reason for attributes change to no-execute?..

Reply:
It changed from no-execute (windows default) to execute in the next commit. https://github.com/jellyfin/jellyfin/pull/508/commits/88c0cf1c5a9783dad093dcba829867bd02a8cbfd

It's just that windows I have to index edit to add execute bit on "new" (or renamed) files.

Reply:
So I wasn't looking at the very recent changes then... sorry!